packages/cli/register/src/ir-to-fdr-converter/convertAuth.ts

```diff
@@ -48,7 +48,9 @@ export function convertAuth(
                                   endpointId: FdrCjsSdk.EndpointId(
                                       scheme.configuration.tokenEndpoint.endpointReference.endpointId
                                   ),
-                                  accessTokenLocator: FdrCjsSdk.JqString(tokenPath)
+                                  accessTokenLocator: FdrCjsSdk.JqString(tokenPath),
+                                  headerName: scheme.configuration.tokenHeader,
+                                  tokenPrefix: scheme.configuration.tokenPrefix
```

Comment:
Can you include a fixture here that tests this code path https://github.com/fern-api/fern/tree/main/packages/cli/register/src/ir-to-fdr-converter/__test__/fixtures (probably one specifically for oauth with custom header)